src/hotspot/share/runtime/arguments.cpp

```diff
@@ -3417,19 +3416,15 @@ bool Arguments::handle_deprecated_print_gc_flags() {
 
 static void apply_debugger_ergo() {
 #ifndef PRODUCT
-  // UseDebuggerErgo is notproduct
```

Comment:
Now that the flag has been changed to a develop flag, it seems wrong that these are guarded by "#ifndef PRODUCT". Shouldn't this be changed to check for ASSERT instead?

Reply:
Yes, ifdef ASSERT is more appropriate.